allofplos/update.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from . import get_corpus_dir, newarticledir, uncorrected_proofs_text_list
```

Comment:
I don't think this will work anymore, you'll need to get those from .corpus, right?